python/tvm/relay/to_onnx.py

```diff
@@ -0,0 +1,409 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, import-self, len-as-condition, unused-argument, too-many-lines, redefined-builtin
+"""Relay to ONNX serialization """
+
+import numpy
+import onnx
+import onnx.utils
+from onnx import numpy_helper
+from tvm.autotvm.graph_tuner.utils.traverse_graph import _expr2graph_impl
+from tvm.relay.expr import Call, TupleGetItem, Var, Constant, Tuple
+
+
+def relay_layout_to_storage_order(storage_order):
+    """converter of tvm storage order format parameter to onnx storage order"""
+    if storage_order not in ('NCHW', 'NHWC'):
+        raise Exception("Mode of storage_order must be either 'NCHW' or 'NHWC'")
+
+    return 0 if storage_order == 'NCHW' else 1
```

Comment:
if possible, just return the storage_order as it is, then we can scale to support more layouts in the future